README.md

```diff
@@ -0,0 +1,33 @@
+# Keytos' RADIUS Client
+
+A RADIUS client that can carry out the following authentication protocols:
+
+- `mab` (MAC address authentication bypass)
+- `pap`
+- `eap-ms-chapv2`
+- `eap-tls`
+- `eap-ttls-pap`
+- `eap-ttls-eap-ms-chapv2`
+- `eap-ttls-eap-tls`
+- `peap-ms-chapv2`
+
+This is used internally at Keytos to thoroughly test [EZRADIUS](https://www.keytos.io/cloud-radius-as-a-service-for-azure-and-entra-id).
+This client seeks to close the gap from existing testing clients such as [`radclient`](https://wiki.freeradius.org/config/Radclient)
+and [`eapol_test`](https://manpages.debian.org/testing/eapoltest/eapol_test.8.en.html),
+by first consolidating the capabilities of both tools into one, and then extending
+the tool to support [RADIUS-over-TLS (RadSec)](https://datatracker.ietf.org/doc/html/rfc6614)
```

Comment:
can you add some sample calls so customers can run it

Reply:
done, have created an issue to add releases for windows. will allow them to just download an exe and run it.